data/mods/MindOverMatter/monsters/feral_lab_psychics.json

```
@@ -0,0 +1,151 @@
+      },
+      {
+        "type": "spell",
+        "spell_data": { "id": "telekinetic_hammer_monster", "min_level": 6 },
+        "cooldown": 12,
+        "monster_message": "%1$s hammers you with psionic force!"
+      },
+      {
+        "id": "smash",
+        "move_cost": 80,
+        "cooldown": 8,
+        "damage_max_instance": [ { "damage_type": "bash", "amount": 20, "armor_penetration": 10 } ],
+        "hitsize_min": 12,
+        "range": 8,
+        "throw_strength": 100,
+        "blockable": false,
+        "effects_require_dmg": false,
+        "hit_dmg_u": "%1$s stares at you and a powerful force hurls you through the air!",
+        "hit_dmg_npc": "%1$s stares at <npcname> and a powerful force hurls them through the air!",
+        "miss_msg_u": "%s stares at you, and you feel a crushing pressure for a moment before the feeling vanishes!",
+        "miss_msg_npc": "%s stares at <npcname> but nothing happens!"
+      },
+      [ "PULL_METAL_WEAPON", 8 ],
+      [ "FEAR_PARALYZE", 10 ]
+    ],
+    "flags": [
+      "SEES",
+      "HEARS",
+      "SMELLS",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "HUMAN",
+      "CAN_OPEN_DOORS",
+      "PATH_AVOID_DANGER_2",
+      "PATH_AVOID_FIRE"
+    ]
+  },
+  {
+    "id": "mon_feral_security_psychic",
+    "type": "MONSTER",
+    "name": "feral security guard, Ψ Division",
+    "description": "A security guard gone feral, or so you thought.  As you look closer, you notice the relatively-normal appearance, the clear eyes, and the patch on their uniform with a Ψ shape in front of the logo of XEDRA.  The zombies around them ignore them as they move smoothly and purposefully through the ruins.",
+    "default_faction": "science",
+    "looks_like": "chud",
+    "bodytype": "human",
+    "species": [ "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 120,
+    "speed": 115,
+    "material": [ "flesh" ],
+    "color": "pink",
+    "symbol": "@",
+    "aggression": 30,
+    "morale": 100,
+    "melee_skill": 7,
+    "melee_dice": 2,
+    "melee_dice_sides": 6,
+    "melee_damage": [ { "damage_type": "bash", "amount": 6 } ],
+    "attack_cost": 85,
+    "weakpoint_sets": [ "wps_humanoid_body" ],
+    "families": [ "prof_intro_biology", "prof_physiology" ],
+    "dodge": 5,
+    "armor": { "bash": 8, "cut": 6, "stab": 6, "bullet": 10 },
+    "harvest": "human",
+    "dissect": "dissect_human_sample_single",
+    "vision_day": 35,
+    "vision_night": 7,
+    "path_settings": { "max_dist": 30, "allow_open_doors": true, "avoid_traps": true, "avoid_sharp": true },
+    "death_drops": "feral_security_death_drops_psychic",
+    "zombify_into": "mon_zombie_survivor",
+    "anger_triggers": [ "FRIEND_DIED", "FRIEND_ATTACKED", "HURT" ],
+    "special_attacks": [
+      { "id": "smash", "attack_upper": true, "throw_strength": 50, "cooldown": 10 },
+      { "id": "bio_op_disarm", "cooldown": 10 },
+      { "id": "bio_op_takedown", "cooldown": 15 }
+    ],
```

Apparently bio_op_disarm is not unhardcoded, but we are in the process, so two ways here: 
First is yeet it and turn back after the jsonization
```suggestion
    "anger_triggers": [ "FRIEND_DIED", "FRIEND_ATTACKED", "HURT" ],
    "//": "TODO: add bio_op_disarm when it would be turned back",
    "special_attacks": [
      { "id": "smash", "attack_upper": true, "throw_strength": 50, "cooldown": 10 },
      { "id": "bio_op_takedown", "cooldown": 15 }
    ],
```

---

or simply left the hardcode version, and left it replacement on the person, that jsonify it
```suggestion
    "anger_triggers": [ "FRIEND_DIED", "FRIEND_ATTACKED", "HURT" ],
    "special_attacks": [
      { "id": "smash", "attack_upper": true, "throw_strength": 50, "cooldown": 10 },
      [ "BIO_OP_DISARM", 10 ],
      { "id": "bio_op_takedown", "cooldown": 15 }
    ],
```

---

I'll go with this one for the moment